includes/gateways/stripe/includes/admin/admin-filters.php

```diff
@@ -34,4 +34,66 @@ function give_stripe_link_transaction_id( $transaction_id, $payment_id ) {
 }
 
 add_filter( 'give_payment_details_transaction_id-stripe', 'give_stripe_link_transaction_id', 10, 2 );
-add_filter( 'give_payment_details_transaction_id-stripe_ach', 'give_stripe_link_transaction_id', 10, 2 );
+add_filter( 'give_payment_details_transaction_id-stripe_checkout', 'give_stripe_link_transaction_id', 10, 2 );
+add_filter( 'give_payment_details_transaction_id-stripe_sepa', 'give_stripe_link_transaction_id', 10, 2 );
+add_filter( 'give_payment_details_transaction_id-stripe_becs', 'give_stripe_link_transaction_id', 10, 2 );
+
+/**
+ * This function is used to add per-form Stripe account management.
+ *
+ * @since 2.7.0
+ *
+ * @param array $settings Settings List.
+ * @param int   $form_id  Form ID.
+ *
+ * @return array
+ */
+function give_stripe_add_metabox_settings( $settings, $form_id ) {
+	$form_account         = give_is_setting_enabled( give_get_meta( $form_id, 'give_stripe_per_form_accounts', true ) );
+	$account_options      = give_stripe_get_account_options();
+	$account_options_keys = array_keys( $account_options );
+	$first_account_key    = is_array( $account_options_keys ) && count( $account_options_keys ) > 0 ? $account_options_keys[0] : [];
```

Comment:
@mehul0810 we can update this code. `array_keys ` will always return `array`.
```suggestion
	$first_account_key    = $account_options_keys ? $account_options_keys[0] : [];
```